tests/solidity/test_close_short.py

```diff
@@ -78,14 +80,24 @@ def verify_close_short(
         time_remaining = 0
         if maturity_time > self.hyperdrive.block_time.time:
             time_remaining = maturity_time - self.hyperdrive.block_time.time
-        self.assertEqual(  # bond reserves
-            self.hyperdrive.market_state.bond_reserves,
-            market_state_before.bond_reserves - time_remaining * bond_amount,
-            msg="bond_reserves is wrong",
+        # self.assertEqual(  # bond reserves
```

Comment:
If we don't need to adhere to this test, just delete it. If we do and we are not, then we should fix it before merging.